nvtabular/torch_dataloader.py

```diff
@@ -60,23 +59,23 @@ def __len__(self):
 
     def __iter__(self):
         for idx in range(0, self.num_samples, self.batch_size):
-            tens = [tensor[idx : idx + self.batch_size] for tensor in self.tensors]
-            yield tens[0], tens[1], tens[2]
-            del tens
+            yield [
```

Comment:
when you run the actual notebooks does this cause any OOM errors especially with bigger batch_sizes? I think thats why I put the `del tens`.

Reply:
I got rid of the temporary variable - so don't need to explicitly clean it up. I *think* this should be good